addons/website_sale/controllers/main.py

```diff
@@ -336,6 +336,33 @@ def cart_update_json(self, product_id, line_id, add_qty=None, set_qty=None, disp
             })
         return value
 
+    @http.route([
+        '/shop/history',
+        '/shop/history/page/<int:page>',
+    ], type='http', auth="user", website=True)
+    def orders_followup(self, page=1, by=5, **post):
```

Comment:
Where does parameter `by` come from?
Method `orders_followup` is overloaded in `website_sale_stock` and `website_sale_digital` with different parameters, is it normal ?